XCUITests/TopTabsTest.swift

```diff
@@ -351,3 +351,21 @@ class TopTabsTestIphone: IphoneOnlyTestCase {
         XCTAssertTrue(app.buttons["TabTrayController.maskButton"].isEnabled)
     }
 }
+
+class TopTabsTestIpad: IpadOnlyTestCase {
+    func testUpdateTabCounter(){
+        
+        app/*@START_MENU_TOKEN@*/.buttons["New Tab"]/*[[".buttons[\"New Tab\"]",".buttons[\"TopTabsViewController.newTabButton\"]"],[[[-1,1],[-1,0]]],[1]]@END_MENU_TOKEN@*/.tap()
```

Comment:
It would be good to have a few comments to explain what the tests does, for example, above this line:
// Opening three tabs by tapping on '+' button